README.md

```diff
@@ -9,28 +9,30 @@ HTTP RESTFul for calling DataWow APIs
 gem 'datawow', '~> 1.3.2'
 ```
 
-#### Generate setting
+### Rails
 
 ```console
 $ rails generate datawow:install
 ```
-The generator will install an initializer which describes ALL of library's configuration options. It is *imperative* that you take a look at it. When you are done, you are ready to call it to any of your codes.
-
-**Note**: Our library work with Rails 4.1 or above
+**Note**: Rails 4.1 or above
 
 # Usage
 
-To call our module use `Datawow` and follow by class of APIs which we're going to explain what API we have
+To call our module use `Datawow` then follow by class of APIs which we're going to explain what API we have
 
 ## Class explanation
+These classes it's a instant method that we have prepared for you. You can use our provider or the recommanded method [read](#dynamically_token_setting) to using our APIs
```

Comment:
These classes are instance methods we have created for you. You can use our provider or the recommended method, [read](#dynamically_token_setting), when using our APIs